packages/zosuss/src/SshBaseHandler.ts

```diff
@@ -58,22 +59,103 @@ export abstract class SshBaseHandler implements ICommandHandler {
     public async process(commandParameters: IHandlerParameters) {
         this.mHandlerParams = commandParameters;
 
-        const sshSessCfgOverride: IOverridePromptConnProps[] = [{
-            propertyName: "privateKey",
-            propertiesOverridden: ["password", "tokenType", "tokenValue", "cert", "certKey", "passphrase"]
-        }];
-        const sshSessCfg: ISshSession = SshSession.createSshSessCfgFromArgs(commandParameters.arguments);
-        const sshSessCfgWithCreds = await ConnectionPropsForSessCfg.addPropsOrPrompt<ISshSession>(
-            sshSessCfg, commandParameters.arguments, {
-                parms: commandParameters,
-                propertyOverrides: sshSessCfgOverride,
-                supportedAuthTypes: [SessConstants.AUTH_TYPE_BASIC]
-            }
+        const sshSessCfgOverride: IOverridePromptConnProps[] = [
+            {
+                propertyName: "privateKey",
+                propertiesOverridden: [
+                    "password",
+                    "tokenType",
+                    "tokenValue",
+                    "cert",
+                    "certKey",
+                ],
+            },
+        ];
+        const sshSessCfg: ISshSession = SshSession.createSshSessCfgFromArgs(
+            commandParameters.arguments
         );
+        let sshSessCfgWithCreds =
+            await ConnectionPropsForSessCfg.addPropsOrPrompt<ISshSession>(
+                sshSessCfg,
+                commandParameters.arguments,
+                {
+                    parms: commandParameters,
+                    propertyOverrides: sshSessCfgOverride,
+                    supportedAuthTypes: [SessConstants.AUTH_TYPE_BASIC],
+                }
+            );
         this.mSession = new SshSession(sshSessCfgWithCreds);
 
         this.mArguments = commandParameters.arguments;
-        await this.processCmd(commandParameters);
+
+        try {
+            await this.processCmd(commandParameters);
+        } catch (e) {
+            this.console.log("Initial key passphrase authentication failed!" + "\n");
```

Comment:
Should we move this `log` statement inside the `if` block, since there are other kinds of errors that may occur?